Qt/InstaDam/Selector/freeDrawErase.h

```diff
@@ -32,7 +32,7 @@ class FreeDrawErase : public FreeDrawSelect {
         UNUSED(widget);
     }
     void toPixmap(QPainter *painter) override {UNUSED(painter);}
-
+    void setOpacity(qreal val) {UNUSED(val);}
```

Comment:
5: Cannot find the User Story that the change implements.

Reply:
Please see the revised user story list here (User Story 37) https://wiki.illinois.edu/wiki/display/CS428SP19/IDM+-+Iteration+2